# Define goal's message
string turtle_name
float32 speed_move
float32 distance
bool isForward
float32 speed_rotate
float32 degrees
bool isClockwise
float32 near_velocity
float32 lar_velocity
---
# Define result's message
---
# Define feedback's message

#http://wiki.ros.org/actionlib
